arxiv/mail/templates/README.md

```diff
@@ -0,0 +1,66 @@
+# arXiv Email Templates
```

Comment:
As someone who doesn't write a lot of HTML e-mails by hand, I find this readme really helpful

Reply:
Oh good! I was hoping to leave a good set of references, because HTML email is so specialized.